routes/#checkout.js

```diff
@@ -179,7 +179,14 @@ module.exports = (checkoutBody, checkoutRespond, storeId) => {
             amount[field] = Math.round(amount[field] * 100) / 100
           }
         }
-        amount.total = Math.round((amount.subtotal + amount.freight - amount.discount) * 100) / 100
+
+        ['extra', 'tax'].forEach(field => {
+          if (!amount[field]) {
+            amount[field] = 0
+          }
+        })
+
```

Comment:
De qualquer forma isso aqui não...
Pouco pra cima aí tem um preset to objeto amount com os fields que vão ser usados, deveria ser adicionado lá (se realmente houver um porque né)

```suggestion
```

Reply:
> De qualquer forma isso aqui não...
> Pouco pra cima aí tem um preset to objeto amount com os fields que vão ser usados, deveria ser adicionado lá (se realmente houver um porque né)

Verdade, coloquei lá.